src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml.cs

```diff
@@ -161,6 +167,43 @@ private void Apply_Click(object sender, RoutedEventArgs e)
             }
         }
 
+        // TODO: Instead of a message box create an overlay on each screen that dissapears after sometime
+        // https://stackoverflow.com/a/14522952
+        private void Apply_identify(object sender, RoutedEventArgs e)
+        {
+            AutoClosingMessageBox.Show("Display 2 - Acer Predator", "", 3000);
+        }
+
+        public class AutoClosingMessageBox

```

Comment:
This stackoverflow provided class isn't appropriate. Consider a scenario in which another app has a similar message box with the same text. You'll send your WM_CLOSE to the wrong window. Why isn't this just a new window you control?